src/docs/product/integrations/notification-incidents/threads/index.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: Threads
+sidebar_order: 1
+description: "Learn more about Sentry's Threads integration and how you can effortlessly send alerts from Sentry to Threads."
+---
+
+The Threads Sentry integration allows you to configure Threads as a destination for issue and metric alerts. Threads will keep you updated on the status of the Sentry issue, and let you know when a metric alert has been triggered and resolved.
+
+This integration is maintained and supported by the company that created it. For more details, check out our [Integration Platform documentation](/product/integrations/integration-platform/).
+
+## Install and Configure
+
+<Note>
+
+Sentry owner, manager, or admin permissions are required to install this integration.
+
+Threads **won't** work with self-hosted Sentry.
+
+</Note>
+
+To add the Threads integration, navigate to **Settings > Integrations > Threads** and click "Install".
+
+![](threads-install.png)
+
+### Alert Rules
+
+The Threads Sentry integration supports [issue alerts](/product/alerts/alert-types/#issue-alerts) and [metric alerts](/product/alerts/alert-types/#metric-alerts). When selecting the [action](/product/alerts/create-alerts/issue-alert-config/#then-conditions-actions), choose "Notify Integration" and then "Threads" for issue alerts, or select "Threads" from the action dropdown for metric alerts. Finally, choose the channel to be notified in:
+
+![](threads-rule.png)
+
+A Threads notification will be created automatically when the alert is triggered.
+
+Threads will notify you when a metric alert has been triggered, and again when it was resolved:
+
+![](threads-metric-alert.png)
+
+### Issue Statuses
```

Comment:
This looks good. My only question is whether this should be an H3 or H2. Like Alert Rules makes sense to me under Install and Configure because you have to configure the alert rules. And then the outcome of that is in that second screenshot. But it seems like we don't really talk about any configuration for issues. If my question isn't making sense, please feel free to ping me.

Reply:
Nah that makes sense, I updated it to an h2.